### PR TITLE
Add additionnal unit tests to csharp wrapper

### PR DESCRIPTION
**Pull request by cbienaime**
- Not integrated to the CI yet.

### CODE REVIEW

**Comment by MathieuMorrissette** [comment]
Unit tests are now integrated into the CI.

File: wrappers/csharp/unit-tests/TestManaged.cs [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+        [TestMethod]
+        public void DecryptWithKeyAsString()
+        {
+            string decryptResultString = Managed.DecryptWithKeyAsUtf8String(TestData.EncryptedData, TestData.BytesTestKey);
+            Assert.AreEqual(decryptResultString, TestData.Base64TestData);
+        }
+
+        [TestMethod]
+        public void DecryptWithPassword()
```
Comment (pdugre) [inline wrappers/csharp/unit-tests/TestManaged.cs]: Rename to DecryptWithPassword2

File: wrappers/csharp/unit-tests/TestManaged.cs [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        {
+            const string encrytedDataAsBase64 = "DQwCAAAAAgDsQkLRs1I3054gNOYP7ifVSpOMFEV8vTfoMuZOWAzbMR2b1QLyIe0/NFNKr8rniijd8PxHv29N";
+            const string password = "testPa$$";
+            byte[] decryptResult = Managed.DecryptWithPassword(encrytedDataAsBase64, password);
+            string decryptResultString = Utils.ByteArrayToUtf8String(decryptResult);
+            Assert.AreEqual(decryptResultString, "test Ciph3rtext");
+        }
+
+        [TestMethod]
+        public void DecryptWithPassword2()
```
Comment (pdugre) [inline wrappers/csharp/unit-tests/TestManaged.cs]: Rename to DecryptWithPassword2_5

File: wrappers/csharp/unit-tests/TestNative.cs [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        [TestMethod]
+        public void Decrypt()
+        {
+            byte[] decryptResult = Native.Decrypt(TestData.EncryptedData, TestData.BytesTestKey);
+            string encodedByteArrayToUtf8String = Utils.ByteArrayToUtf8String(decryptResult);
+            Assert.AreEqual(encodedByteArrayToUtf8String, TestData.Base64TestData);
+        }
+
+        [TestMethod]
+        public void Decrypt2()
```
Comment (pdugre) [inline wrappers/csharp/unit-tests/TestNative.cs]: Rename to Decrypt1

File: wrappers/csharp/unit-tests/TestNative.cs [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+                "DQwCAAAAAQCK1twEut+TeJfFbTWCRgHjyS6bOPOZUEQAeBtSFFRl2jHggM/34n68zIZWGbsZHkufVzU6mTN5N2Dx9bTplrycv5eNVevT4P9FdVHJ751D+A==");
+            byte[] encryptKey = Utils.Base64StringToByteArray("ozJVEme4+5e/4NG3C+Rl26GQbGWAqGc0QPX8/1xvaFM=");
+
+            byte[] decryptResult = Native.Decrypt(encryptedData, encryptKey);
+            string decryptResultAsUtf8String = Utils.ByteArrayToUtf8String(decryptResult);
+            Assert.AreEqual(decryptResultAsUtf8String, "test Ciph3rtext~");
+        }
+
+        [TestMethod]
+        public void Decrypt3()
```
Comment (pdugre) [inline wrappers/csharp/unit-tests/TestNative.cs]: Rename to Decrypt2_5

**Pull request merged**